.github/workflows/pull-request-test.yml

```diff
@@ -86,20 +86,26 @@ jobs:
         KUBECONFIG: ${{ github.workspace }}/renkubot-kube.config
         RENKUBOT_KUBECONFIG: ${{ secrets.RENKUBOT_KUBECONFIG }}
         CI_RENKU_VALUES: ${{ secrets.CI_RENKU_VALUES }}
+        RENKUBOT_RANCHER_APISECRET: ${{ secrets.RENKUBOT_RANCHER_APISECRET }}
       run: |
         echo "$RENKUBOT_KUBECONFIG" > renkubot-kube.config
-        helm init --client-only
+        helm repo add stable 	https://kubernetes-charts.storage.googleapis.com
         helm repo add renku https://swissdatasciencecenter.github.io/helm-charts/
         helm repo add gitlab https://charts.gitlab.io/
         helm repo add jupyterhub https://jupyterhub.github.io/helm-chart
         rm -rf charts/renku/charts
         helm dependency update charts/renku
         printf "%s" "$CI_RENKU_VALUES" | sed "s/<replace>/${RENKU_RELEASE}/" > values.yaml
-        kubectl create namespace $RENKU_NAMESPACE || true
+        curl -u "token-fxgsb:$RENKUBOT_RANCHER_APISECRET" \
+        -X POST \
+        -H 'Accept: application/json' \
+        -H 'Content-Type: application/json' \
+        -d '{"containerDefaultResourceLimit":null, "name":${RENKU_RELEASE}, "projectId":"c-l6jt4:p-rz88m", "resourceQuota":null}' \
```

Comment:
Guess it should be `RENKU_NAMESPACE` here, otherwise looks good! 👍

Reply:
this is not yet ready for review

Reply:
oops sorry, misinterpreted the state of the PR

Reply:
yep, I did not put WIP or similar because I wanted the github actions to run. I also need to test the teardown.